runner/update-status

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+if [[ ${1:-} == '' ]]; then
+  # shellcheck source=runner/logger.bash
+  source logger.bash
+  log.error "Missing required argument -- '<phase>'"
+  exit 64
+fi
+
+if [ "${RUNNER_STATUS_UPDATE_HOOK:-false}" == "true" ]; then
+
+    apiserver=https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}
+    serviceaccount=/var/run/secrets/kubernetes.io/serviceaccount
+    namespace=$(cat ${serviceaccount}/namespace)
+    token=$(cat ${serviceaccount}/token)
+
+    jq -n --arg phase "$1" --arg message "$2" '.status.phase = $phase | .status.message = $message' | HTTPS_PROXY= http_proxy= curl \
```

Comment:
The second argument is optional, we have to handle this.

```suggestion
    jq -n --arg phase "$1" --arg message "${2:-}" '.status.phase = $phase | .status.message = $message' | HTTPS_PROXY= http_proxy= curl \
```

We could also treat all arguments that follow as the message:

```suggestion
    phase=$1
    shift
    jq -n --arg phase "$phase" --arg message "${*:-}" '.status.phase = $phase | .status.message = $message' | HTTPS_PROXY= http_proxy= curl \
```